### PR TITLE
Remove Clothesmate Bloat

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

The original intended purpose of Loadouts as designed by Death was to serve as the ultimate solution to the Clothesmate BLOAT. Ironically, when Wizden made their own Loadouts, they made a version of Loadout that is incapable of this. Now that we've more significantly fleshed out EE Loadouts to fulfill that purpose, I can finally trim down the Clothesmate. I haven't touched any of the other vendors, just the Clothesmate. 

As an aside, the cost of a Clothing restock has gone down from a fucking ridiculous 4500, down to only 1500. Making it something Cargo techs can actually reasonably buy. Clothesmates now only contain the absolute barest minimum clothes for a greytider. This also helpfully serves as a balancing act against Loadouts being able to provide items other than clothes. It is much harder to get clothing outside of loadouts, preventing people from spamming useless clothing entities on the station. If you spend all your loadout points on items, you are locking yourself out of wearing actually nice drip. Choose how you spend your points wisely!

<details><summary><h1>Media</h1></summary>
<p>

image

</p>
</details>

---

# Changelog

:cl:
- tweak: Due to budget cuts, Nanotrasen has ceased stocking Clothesmate vendors with more clothing than the average cargo tech can afford. Civilians are advised to bring their own clothes to the station if they wish to wear anything other than grey.

### CODE REVIEW

**Comment by VMSolidus** [comment]
Oh that's right, somewhere in the list is another clothing vendor that contains 141 total items...

**Comment by VMSolidus** [comment]
At some point I'd like to de-bloat some of the other clothing vendors, so that we can get the clothes restock box down to 1000 spesos. 1500 is still completely ridiculous for a box of clothes, and there's other vendors that also have too much bloat.

**Review by OldDanceJacket — APPROVED** [review]

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml [hunk 1]
```diff
@@ -1,116 +1,24 @@
 - type: vendingMachineInventory
   id: ClothesMateInventory
   startingInventory:
-    ClothingBackpack: 5
-    ClothingBackpackDuffel: 5
-    ClothingBackpackSatchel: 3
-    ClothingBackpackSatchelLeather: 2
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml:7]: Keep this backpack and then the PR is fine; just to allow for maximum storage customisation :)

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml (hunk 1 above)
Comment (VMSolidus) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml:7]: Done.

**Review by DangerRevolution — APPROVED** [review]

**Pull request merged**